shepherd/LCM.py

```diff
@@ -1,26 +1,23 @@
 import threading
-import lcm # pylint: disable=import-error
+import json
+import lcm
+
+LCM_address = 'udpm://239.255.76.68:7667?ttl=2'
```

Comment:
when was this added and is it necessary?

Reply:
You need an address to send messages over ethernet/through a switch. If these methods aren't used for that then it isn't necessary